resume.tex

```diff
@@ -24,26 +24,23 @@
 %==============================================================================
 
 \resumeSection{Work Experience}{
-  \experience{Software Development Engineer II}{MetLife Inc.}{2017-Present}{
-    \item Led a small team in building a proof-of-concept for a ``next-phase`` application built on Spring Boot and Angular 4, which was in turn used as a prototype for new projects company-wide
-	\item Consulted for a team building an application using my prototype as a template for a MongoDB-based plan repository with a Material Design UI
-	\item Implemented a report automation system through which users customize scheduled reports for the underwriting team
-    \item Enforced best practices and design principles by conducting code reviews
-  }
-  
+  \experienceHeader{MetLife Tech University Professional Program}{MetLife Inc.}{2014}\\
+  \experienceHeader{Associate Software Development Engineer}{MetLife Inc.}{2015-2016}\\
   \experienceHeader{Software Development Engineer I}{MetLife Inc.}{2016-2017}\\
-  \experience{Associate Software Development Engineer}{MetLife Inc.}{2015-2016}{
+  \experience{Software Development Engineer II}{MetLife Inc.}{2017-Present}{
+    \item Achieved the only perfect score for Technical Aptitude on my Tech University Professional Program team
```

Comment:
Moved the MTUPP projects up here too, because they seem like pretty nice accomplishments.